### PR TITLE
installation instructions and scripts

### PR DESCRIPTION
**Pull request by kfir4444**
including Installation instructions, updated [README.md](https://github.com/DanaResearchGroup/ARC_T3_Workshop/blob/installation/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
-This repository contains scripts and documaentation material for the ARC and T3 workshop
+Here are the files used to automatically install the software required for the workshop.
+
+# installations: 
+
+In your Ubuntu virtual machine, create a "Code" directory using the Files application by right clicking on the background, choosing "New Folder" and choose the name "Code".
+
+Next, open a terminal in the that folder.
+
+This can be done by right clicking on the file explorrer and choosing "Open In Terminal", or using Ctrl+Alt+T and `cd Code`
```
Comment (alongd) [inline README.md]: explorrer typo

File: README.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
-## ARC and T3 Workshop material
+# ARC and T3 Workshop:
+Welcome to the instructions for the ACS fall 2022- ARC and T3 workshop!
 
-This repository contains scripts and documaentation material for the ARC and T3 workshop
+Here are the files used to automatically install the software required for the workshop.
+
+# installations: 
+
+In your Ubuntu virtual machine, create a "Code" directory using the Files application by right clicking on the background, choosing "New Folder" and choose the name "Code".
```
Comment (alongd) [inline README.md]: would you like to say that this should be under "Home"?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+
+Next, open a terminal in the that folder.
+
+This can be done by right clicking on the file explorrer and choosing "Open In Terminal", or using Ctrl+Alt+T and `cd Code`
+
+Run:
+
+`sudo apt install git`
```
Comment (alongd) [inline README.md]: We have this under install_conda.sh

Where do we need it, here or there?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
+
+Then:
+
+`git clone https://github.com/DanaResearchGroup/ARC_T3_Workshop`
+
+Next:
+
+`bash ARC_T3_Workshop/install_conda.sh`
+
+Look for the required interaction while installing. for example, you may need to type "yes" in the terminal to approve the terms and conditions.
```
Comment (alongd) [inline README.md]: approve the terms and conditions for Anaconda.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+
+Make sure that you are installing Anaconda3 under the default prefix `"home/username/anaconda3"`.
+
+Then, close the terminal, and open a new one in the same directory.
+
+Run:
+
+`bash ARC_T3_Workshop/install_mamba.sh`
```
Comment (alongd) [inline README.md]: can't we merge them, and install mamba in our script?

File: README.md [hunk 6]
```diff
@@ -1,3 +1,58 @@
...
+
+`bash ARC_T3_Workshop/make_envs.sh`
+
+# Optional: Testing
+
+After the installation is finished, navigate (using the terminal of the Files application) to `RMG-Py` (in terminal: `cd ~/Code/RMG-Py`), then, in the terminal, write the following command:
```
Comment (alongd) [inline README.md:45]: perhaps say that this step could take a while (~30 min?)

File: README.md [hunk 7]
```diff
@@ -1,3 +1,58 @@
...
+in the terminal 
+`cd ../ARC`
+
+then
+
+`conda activate arc_env`
+
+`make test`
+
+
```
Comment (alongd) [inline README.md:63]: Users would expect to know what they should see after the tests so they can know that they passed w/o errors

File: make_envs.sh [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+echo 'export PATH=$HOME/Code/ARC/:$PATH' >> ~/.bashrc
+
+echo 'export PYTHONPATH=$HOME/Code/T3/:$PYTHONPATH' >> ~/.bashrc
+echo 'export PATH=$HOME/Code/T3/:$PATH' >> ~/.bashrc
+
+source ~/.bashrc
+
+conda activate rmg_env
+cd RMG-Py/
+make
```
Comment (alongd) [inline make_envs.sh]: does this work now?

**Review by alongd — COMMENTED** [review]
Perfect, thanks!!
I added some comments, please take a look

File: README.md (hunk 2 above)
Comment (kfir4444) [reply README.md]: It's the default when opening the Files and the terminal.. I can add something about it though

File: README.md (hunk 3 above)
Comment (kfir4444) [reply README.md]: It's here because we use `git clone` for the repo, we probably don't need it at all anyways, it's for security.

Comment (alongd) [reply README.md]: why have it twice? perhaps put the full command here, and remove it from the script?

**Review by alongd — APPROVED** [review]
Thanks!!

**Pull request merged**